Source/ASVideoNode.h

```diff
@@ -147,6 +147,13 @@ NS_ASSUME_NONNULL_BEGIN
  * @param videoNode The videoNode
  */
 - (void)videoNodeDidRecoverFromStall:(ASVideoNode *)videoNode;
+/**
+ * @abstract Delegate method invoked when an error occurs while trying to play a video
+ * @param videoNode The videoNode.
+ * @param currentItem The error that occurs
+ */
+- (void)videoNodeDidFailToInitAssetFor:(ASVideoNode *)videoNode withError:(NSError *)error;
```

Comment:
To follow convention, let's make it `-(void)videoNode:(ASVideoNode *)videoNode didFailToLoadAssetWithError:(NSError *)error`.